core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -421,12 +425,18 @@ public ContainerisedDockerCompose(List<File> composeFiles, String identifier) {
         //  as the docker daemon, just mapping the docker control socket is OK.
         // As there seems to be a problem with mapping to the /var/run directory in certain environments (e.g. CircleCI)
         //  we map the socket file outside of /var/run, as just /docker.sock
-        addFileSystemBind("/var/run/docker.sock", "/docker.sock", READ_WRITE);
+        addFileSystemBind(getDockerSocketHostPath(), "/docker.sock", READ_WRITE);
         addEnv("DOCKER_HOST", "unix:///docker.sock");
         setStartupCheckStrategy(new IndefiniteWaitOneShotStartupCheckStrategy());
         setWorkingDirectory(containerPwd);
     }
 
+    private String getDockerSocketHostPath() {
+        return SystemUtils.IS_OS_WINDOWS
```

Comment:
I think this is making an assumption that we are only using docker via their sockets interface.  However in Docker Machine setups this may not be available  It should be able to use the `DOCKER_HOST`and `DOCKER_CERT_PATH`  and `DOCKER_TLS` values.